.github/workflows/_e2e.yml

```
@@ -101,11 +103,11 @@ jobs:
           retention-days: 30
 
       - name: Build Kiosk
-        if: inputs.isRpc || inputs.isTypescriptSDK
+        if: inputs.isRpc || inputs.isTypescriptSDK || github.ref_name == 'develop'
```

imo doesnt make sense to keep `isRpc` only for these 2 tests
```suggestion
        if: inputs.isTypescriptSDK || github.ref_name == 'develop'
```

---

What do you mean by `only` ? `isRpc` is used more times across the file

---

right right